build.gradle.kts

```diff
@@ -511,6 +511,7 @@ allprojects {
                     exceptionFormat = TestExceptionFormat.FULL
                     showStandardStreams = true
                 }
+                include("**/RedisServer*")
```

Comment:
I would suggest refrain from putting `redis`-specific configuration to a common code path.

Reply:
Ok, Let me change it